File: app/src/main/java/com/example/watering/investrecord/DBHelper/SpendDBHelper.java
package com.example.watering.investrecord.DBHelper;

import android.content.Context;

/**
 * Created by watering on 17. 10. 21.
 */

@SuppressWarnings("DefaultFileTemplate")
public class SpendDBHelper extends DBHelper {

    public SpendDBHelper(Context context) {
        super(context);

        TABLE_NAME = "tbl_spend";
        COLUMNS = new String [] {"spend_code TEXT UNIQUE","details TEXT","id_sub INTEGER","date_use TEXT","amount INTEGER"};
    }
}

File: app/src/main/java/com/example/watering/investrecord/dbhelper/SpendDBHelper.kt
package com.example.watering.investrecord.dbhelper

import android.content.Context

/**
 * Created by watering on 17. 10. 21.
 */

class SpendDBHelper(context: Context) : DBHelper(context) {

    init {

        TABLE_NAME = "tbl_spend"
        COLUMNS = arrayOf("spend_code TEXT UNIQUE", "details TEXT", "id_sub INTEGER", "date_use TEXT", "amount INTEGER")
    }
}
